src/unit-tests/osloader-test/ut_module.c

```diff
@@ -3,6 +3,7 @@ void MODULE_NAME(void)
   volatile int i;
   i = 1; 
   i = i + 1;  /* squelch set-but-not-used compiler warning */
+  (void)i;
```

Comment:
What's this for?

Reply:
It just quiets down a warning from cppcheck that "i" was set but not used.  The statement before it (the increment) worked to avoid a gcc warning about the same but cppcheck still complained.

Note that as part of the testing I did run this through cppcheck and resolve any remaining issues (should have noted that in the pull request)